data/data/aws/vpc/sg-master.tf

```diff
@@ -56,6 +56,26 @@ resource "aws_security_group_rule" "master_ingress_https" {
   to_port     = 6443
 }
 
+resource "aws_security_group_rule" "master_self_icmp" {
```

Comment:
question: why is the self required?

Reply:
this is the same as the vxlan self rule - need to allow icmp intra-vpc.